word_guess.rb

```diff
@@ -0,0 +1,192 @@
+                   ______  (@|   "+"*****".colorize(:light_yellow)+"**  ****".colorize(:light_red)+"         ***".colorize(:red)+"   _  _   º "+"**".colorize(:red)+"
+                  /"+"******".colorize(:light_yellow)+"~~~~||"+"*****".colorize(:light_yellow)+"******".colorize(:light_red)+"*********  ******".colorize(:red)+" (.|( ) ✨
+          |       `--____"+"**********".colorize(:light_yellow)+"*********".colorize(:light_red)+"******** ******".colorize(:red)+"  (_Y|_/ "+"** **".colorize(:red)+"
+         / ~~~--_____    ~~~/ "+"******".colorize(:light_yellow)+"**********".colorize(:light_red)+"*********** **".colorize(:red)+"  /_| "+"**      **".colorize(:red)+"
+                     `~~~~~         "+"***".colorize(:light_yellow)+"*********".colorize(:light_red)+"*********".colorize(:red)+"       L    "+" ***".colorize(:red)+"
+                                          **".colorize(:light_yellow)+"**    *****".colorize(:light_red)+"  ***** ****".colorize(:red)+"
+                                            "+"***".colorize(:red)+"
+
+    "
+    elsif @tally == 4
+    puts "
+    <~>                          ___________
+     | |,_____                  (BWAHAHAAHA!)
+           ___`|                 ⁻⁻⁻⁻⁻⁻⁻⁻⁻⁻⁻
+           |"+"('>".colorize(:red)+"|`-__            o
+             ~      ~~~--__   º        "+"**              ***".colorize(:light_red).blink+"
+                   ______  (@|   "+"*".colorize(:light_yellow).blink+"******  *".colorize(:light_red).blink+"***    *******    ******".colorize(:red).blink+"
+                  /"+"******".colorize(:light_yellow).blink+"~~~~||"+"*".colorize(:light_yellow).blink+"***********".colorize(:light_red).blink+"**********************".colorize(:red).blink+"
+          |       `--____"+"*******".colorize(:light_yellow).blink+"******".colorize(:light_red).blink+"*****************************".colorize(:red).blink+"
+         / ~~~--_____    ~~~/ "+"***".colorize(:light_yellow).blink+"******".colorize(:light_red).blink+"******************************".colorize(:red).blink+"
+                     `~~~~~         "+"*".colorize(:light_yellow).blink+"*********".colorize(:light_red).blink+"********************".colorize(:red).blink+"
+                                          "+"****    *********".colorize(:light_red).blink+"*****".colorize(:red).blink+"
+                                            "+"***       ********".colorize(:light_red).blink+"***".colorize(:red).blink+"
+                                                            "+"****".colorize(:light_red).blink+"****".colorize(:red).blink+"
+
+    ".colorize(:red).blink
+    puts "The word was: #{@current_word}".colorize(:magenta)
+    end
+  end
+
+def check_letter
+  #we ask for user input using gets.chomp
+  @tally = 0
+  while @tally < 4
+    puts "Choose a letter at your own risk!"
+    puts "Word: #{@correct_letters.join(" ")}".colorize(:green)
+    letter = gets.chomp
+    if letter.length > 1
+      if letter == @current_word
```

Comment:
Since these two if statements must be true in order to execute the `you_win` and `exit`, and there is no `else` on the second `if` statement. I would combine these two into one like:
`if (letter.length > 1) && (letter == @current_word)`